pkg/runner/test_runner.go

```diff
@@ -137,11 +139,11 @@ func (tr *TestRunner) Run(
 	}
 
 	emitterForStep := make(map[string]testevent.Emitter)
-	for _, sb := range t.TestStepsBundles {
+	for _, sb := range stepBundlesToRun {
 		emitterForStep[sb.TestStepLabel] = emitterFactory.New(sb.TestStepLabel)
 	}
 
-	stepOutputs, err := newTestStepsVariables(t.TestStepsBundles, func(tgtID string, stepLabel string, name string, value json.RawMessage) {
+	onVariableAdded := func(tgtID string, stepLabel string, name string, value json.RawMessage) {
```

Comment:
```suggestion
	onStepOutputAdded := func(tgtID string, stepLabel string, name string, value json.RawMessage) {
```

this was supposed to be named "step outputs" in the first place